frontend/sozisel-app/src/components/SessionDetails/EditSession/EditSession.tsx

```diff
@@ -0,0 +1,133 @@
+import "./EditSession.scss";
+
+import MuiAlert, { AlertProps } from "@material-ui/lab/Alert";
+import { ReactElement, useState } from "react";
+import {
+  useSessionDetailsQuery,
+  useUpdateSessionMutation,
+} from "../../../graphql";
+
+import CircularProgress from "@material-ui/core/CircularProgress";
+import MainNavbar from "../../MainNavbar/MainNavbar";
+import SessionCustomization from "../SessionCustomization/SessionCustomization";
+import Snackbar from "@material-ui/core/Snackbar";
+import TemplateOverview from "../TemplateOverview/TemplateOverview";
+import { useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+function Alert(props: AlertProps) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+export interface SessionEditProps {
+  sessionId: string;
+}
+
+export default function EditSession(): ReactElement {
+  const { t } = useTranslation("common");
+  const location = useLocation();
+  const [successMessage, setSuccessMessage] = useState<string>("");
+  const { data, loading, error } = useSessionDetailsQuery({
+    fetchPolicy: "network-only",
+    variables: {
+      id: (location.state as SessionEditProps).sessionId,
```

Comment:
I think that the url should contain the session id itself as an url param. Then could use some react router's functionality to fetch that param.